packages/experiment-tag/src/experiment.ts

```diff
@@ -186,6 +189,59 @@ const revertMutations = () => {
   }
 };
 
+const inject = (js: string, html, utils, id) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  eval(js, html, utils, id);
```

Comment:
pass these fields to eval to make it so they're not stripped out by the transpiler